### DISCUSSION TITLE
PodcastSeries episodes

### DISCUSSION THREAD

**Discussion by tttmabo** [schemaorg/schemaorg#3108] [Q&A]
Does the PodcastSeries object have a place for episodes in the way an Event has subEvent? Or do we simply implement the Episode page with a partOfSeries and allow indexing systems to figure it out on their own?

**Comment by chrisspradling1980** [comment]
Although I do not know the answer to this question, I will say that it is
an excellent question. Sub events and sub domains do not have enough
representation because they are so hard to configure as a property.

**Comment by RichardWallis** [comment]
As per the [example](https://schema.org/PodcastEpisode#eg-0307) on the [PodcastEpisode](https://schema.org/PodcastEpisode) page, the relationship between an episode and its series is derived via the [partOfSeries](https://schema.org/partOfSeries) property.  You also have the option of referencing a season it is part of using [partOfSeason](https://schema.org/partOfSeason).

Providing the referenced entities are available to those consuming the data, that should be sufficient.

If you did want to explicitly reference episodes from within the PodcastSeries description, you could use the [hasPart](https://schema.org/hasPart) property.

**Reply by tttmabo** [reply]
Perfect. This is exactly what I was looking for. Thank you!
